clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -292,6 +295,7 @@ export default async function handler(
     `
         : ""
     }
+  window.Fides.version = ${JSON.stringify(gitTagVersion().split("-")[0])};
```

Comment:
Does this work in a built image, though? Once this is dockerized and running on an image the git metadata is probably not reliable?

Reply:
Good point!